src/pipecat/services/assemblyai/stt.py

```diff
@@ -190,13 +190,17 @@ async def _receive_task_handler(self):
             while self._connected:
                 try:
                     message = await self._websocket.recv()
+                    self.start_watchdog()
                     data = json.loads(message)
                     await self._handle_message(data)
+                    self.reset_watchdog()
```

Comment:
I guess since we also have it inside the `finally`, we can remove this one.

Reply:
Yep!